tests/helpers/providerStateHelper.js

```diff
@@ -33,7 +33,7 @@ const givenGroupExists = (provider, group) => {
  */
 const givenFileExists = (provider, username, password, resource) => {
   return provider
-    .given('file exists', { username, password, fileName: encodeURIComponent(resource) })
+    .given('file exists', { username, password, fileName: encodeURIComponent(resource).split('%2F').join('/') })
```

Comment:
use it in a function like in https://github.com/owncloud/owncloud-sdk/pull/801/files#diff-a16db6b5aac8f41269353f9be2487f36a04fd3e16286cd2c0edf4ce46a489aa5R24